.github/workflows/ci-python-sdk.yaml

```diff
@@ -311,6 +311,39 @@ jobs:
       SNOWFLAKE_ACCOUNT_NAME: ${{ secrets.SNOWFLAKE_UNAME }}
       SNOWFLAKE_PASSWORD: ${{ secrets.SNOWFLAKE_PASSWORD }}
 
+  Generate-Constraints:
```

Comment:
@kaxil why didn't we use the Airflow's constraint files?

Reply:
One of the main reasons is that we have additional dependencies than Airflow, which increases the possibility of dependency conflicts as seen in https://github.com/astronomer/astro-sdk/issues/1226

Reply:
yeah, I had a difficult time making it work with airflow constraint files